detekt.yml

```diff
@@ -0,0 +1,785 @@
+build:
+  maxIssues: 0
+  excludeCorrectable: false
+  weights:
+    # complexity: 2
+    # LongParameterList: 1
+    # style: 1
+    # comments: 1
+
+config:
+  validation: true
+  warningsAsErrors: false
```

Comment:
Only thing I'd suggest is turning this `true`, otherwise warnings pile up with no urgency to fix them.